db/schema.rb

```diff
@@ -48,11 +54,26 @@
     t.string "shelter_type"
   end
 
+  create_table "bus_stops_field_names", id: false, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
This seems like an accidental inclusion?

Reply:
tis